src/screens/authentication/Login/index.js

```diff
@@ -20,11 +21,15 @@ const onGoogleButtonPress = async () => {
     const userEmail = signInData.user.email
     if (await AuthenticationService.emailAlreadyHasGoogleSignIn(userEmail)) {
       await AuthenticationService.loginWithGoogleSignInData(signInData)
+      await MessagingService.checkForPermissions()
+      await MessagingService.saveFCMTokenForCurrentUser()
```

Comment:
Do any of these functions throw an error? If they do, extract these two lines to a function and surround them with a try/catch, so that we can safely ignore the error and the login flow continues as normal.